.travis.yml

```diff
@@ -21,9 +21,15 @@ matrix:
     # Linux environment to test scikit-learn against numpy and scipy master
     # installed from their CI wheels in a virtualenv with the Python
     # interpreter provided by travis.
-    -  python: 3.7
-       env: CHECK_WARNINGS="true"
-       if: type = cron OR commit_message =~ /\[scipy-dev\]/
+    - python: 3.7
+      env: CHECK_WARNINGS="true"
+      if: type = cron OR commit_message =~ /\[scipy-dev\]/
+    
+    - python: 3.7
+      env:
+        - CHECK_WARNING="true"
+        - BUILD_WITH_ICC="true"
+      if: type = cron OR commit_message =~ /\[scipy-dev\]/
```

Comment:
Do we want to use the same scipy-dev tag for this?

Reply:
I put the same name to avoid having options that we will not remember :)
But I'm ok to put a different name. `icc-build` ?

Reply:
`icc` or `icc-build` both are fine to me.